app/json/slate_config.schema.json

```diff
@@ -36,6 +36,16 @@
         ],
         "pattern": "^(.+)$"
       },
+      "curatorTopicLabel": {
```

Comment:
Consider adding a enum so we don't enter invalid topics: https://json-schema.org/understanding-json-schema/reference/generic.html#enumerated-values